Makefile

```diff
@@ -0,0 +1,22 @@
+PREFIX ?= $(shell pwd)
+
+all: format build
+
+format:
+	@echo ">> formatting code"
+	@go fmt ./...
```

Comment:
Literally cannot get it to run (: But I guess leaving gofmt here is fine since missing or extra imports will cause things to not compile anyway, which CI will catch.

Reply:
yeah I was thinking about goimports just in terms of sorting imports (: